EstateManagement/Controllers/ContractController.cs

```diff
@@ -209,6 +211,43 @@ public class ContractController : ControllerBase
                                 });
         }
 
+        [HttpPost]
+        [Route("{contractId}/products/{productId}/transactionFees/{transactionFeeId}")]
+        public async Task<IActionResult> DisableTransactionFeeForProduct([FromRoute] Guid estateId,
+                                                                               [FromRoute] Guid contractId,
+                                                                               [FromRoute] Guid productId,
+                                                                               [FromRoute] Guid transactionFeeId,
+                                                                               CancellationToken cancellationToken)
+        {
+            // Get the Estate Id claim from the user
+            Claim estateIdClaim = ClaimsHelper.GetUserClaim(this.User, "EstateId", estateId.ToString());
+
+            String estateRoleName = Environment.GetEnvironmentVariable("EstateRoleName");
+            if (ClaimsHelper.IsUserRolesValid(this.User, new[] { string.IsNullOrEmpty(estateRoleName) ? "Estate" : estateRoleName }) == false)
+            {
+                return this.Forbid();
+            }
+
+            if (ClaimsHelper.ValidateRouteParameter(estateId, estateIdClaim) == false)
+            {
+                return this.Forbid();
+            }
+
+            //// Create the command
+            //AddTransactionFeeForProductToContractRequest command = AddTransactionFeeForProductToContractRequest.Create(contractId, estateId,
+            //                                                                                                           productId,
+            //                                                                                                           transactionFeeId,
+            //                                                                                                           addTransactionFeeForProductToContractRequest.Description,
+            //                                                                                                           calculationType,
+            //                                                                                                           addTransactionFeeForProductToContractRequest.Value);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5995655)